README.md

```diff
@@ -31,34 +32,32 @@ The core AWS components used by this deployment include the following AWS servic
 
 # Regional Availability
 
-Custom resource auto scaling is available in Canada (Central), US West (N. California), US East (N. Virginia), US East (Ohio), US West (Oregon), South America (Sao Paulo), EU (Frankfurt), EU (Ireland), EU (London), EU (Paris), Asia Pacific (Mumbai), Asia Pacific (Seoul), Asia Pacific (Singapore), Asia Pacific (Sydney), Asia Pacific (Tokyo), and China (Beijing). 
+Custom resource auto scaling is available in Canada (Central), US West (N. California), US East (N. Virginia), US East (Ohio), US West (Oregon), South America (Sao Paulo), EU (Frankfurt), EU (Ireland), EU (London), EU (Paris), Asia Pacific (Mumbai), Asia Pacific (Seoul), Asia Pacific (Singapore), Asia Pacific (Sydney), Asia Pacific (Tokyo), and China (Beijing).
 
-# Prerequisites
+# First Steps
 
-* AWS Command Line Interface [installed](https://docs.aws.amazon.com/cli/latest/userguide/installing.html) and [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html). 
-* User credentials with permissions that allow you to configure automatic scaling and create the required service-linked role. For more information, see the [Application Auto Scaling User Guide](https://docs.aws.amazon.com/autoscaling/application/userguide/auth-and-access-control.html).  
-* Permissions to create a stack using a CloudFormation template, plus full access permissions to resources within the stack. For more information, see the [AWS CloudFormation User Guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html). 
-* Permissions to [publish metrics (PutMetricData)](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/permissions-reference-cw.html) to CloudWatch.
```

Comment:
any idea why this is moved to later section

Reply:
The permissions that relate to configuring scaling policies are listed right at the point before the policy is configured. If the person who originally sets everything up delegates responsibility to manage the scaling policies to someone else, then that person does not have to worry about the stack permissions. It's a good practice to keep permissions close to the API operations they relate to, as we can never know when someone else might need to manage a specific part of the configuration in an end-to-end solution. If they run into problems, they should be able to easily check whether their permissions are the problem.